src/Pages/MyCollection/MyCollection.module.css

```diff
@@ -0,0 +1,18 @@
+.myCollectionPage {
+  width: 100vw;
+  display: grid;
+  place-items: center;
+  grid-template-rows: 8rem 4rem 17rem auto;
+  overflow: scroll;
+  padding-bottom: 4.5rem;
+}
+
+.myCollectionPage::-webkit-scrollbar {
+  display: none;
+}
+
+.navBar {
+  width: 100vw;
```

Comment:
I don't like the `vw` here because this is a child element of a page, which has a size set via the viewport. Child elements shouldn't ignore their parent element's sizes.